File: programme/multiplication-table/table.cpp
// Write a programme to output the multiplication table
#include <iostream>

using namespace std;

int main()
{
    int n = 5, i;
    cout << "The multiplication table for " << n << " is as follows:" << endl;
    for (i = 1; i <= 10; i++)
        cout << n << " * " << i << " = " << n * i << endl;
    return 0;
}

File: programme/multiplication-table/table.js
// Write a programmeto output the multiplication table

"use strict";

// perform following command to run programme
// node table.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

function table_of(n) {
  for (let i = 1; i <= 10; i++) {
    console.log(`${i} x ${n} = ${i * n}`);
  }
}

rl.question("Input  : ", (n) => {
  console.log("Output : ");
  n = parseInt(n);
  if (isNaN(n)) {
    console.error("Some error occured");
  } else {
    table_of(n);
  }
  rl.close();
});
